uber/templates/base.html

```diff
@@ -45,6 +45,27 @@
     </style>
     <script>
         var DISABLE_STRIPE_BUTTONS_ON_CLICK = true;
```

Comment:
This is the same idea behind how we do ``BADGE_TYPES`` for the prereg forms.  There's a global Javascript object which can be added to or otherwise modified to add new sections and items to the main admin menu.